Sources/Common/Logging.swift

```diff
@@ -41,19 +41,21 @@ extension OSLog {
         case remoteMessaging = "Remote Messaging"
         case subscription = "Subscription"
         case history = "History"
+        case autofill = "Autofill"
     }
 
 #if DEBUG
     // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
     // To activate Logging Categories for DEBUG add categories here:
-    static var debugCategories: Set<Categories> = [ /* .userScripts, */ ]
+    static var debugCategories: Set<Categories> = [ .autofill ]
```

Comment:
This will be reverted before merging, right?

Reply:
Done